tests/test_parser.py

```diff
@@ -500,3 +499,172 @@ def test_path_and_fileobj_together(self):
     def test_nonseekable_fileobj(self):
         with pytest.raises(AttributeError):
             DockerfileParser(fileobj=sys.stdin)
+
+    @pytest.mark.parametrize('instruction', [
+        'LABEL',
+        'ENV'
+    ])
+    def test_context_structure_per_line(self, dfparser, instruction):
+        dfparser.content = """
```

Comment:
Thank you, don't know it.

I only need to add extra space after backslash in `{0} description="This text illustrates \ ` here:

```python
        dfparser.content = (dedent("""
            FROM fedora:25
:
:
            {0} description="This text illustrates \ 
            that label-values can span multiple lines."
            """)).format(instruction)
```